internal/hcsoci/pnp.go

```diff
@@ -0,0 +1,44 @@
+// +build windows
+
+package hcsoci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Microsoft/hcsshim/internal/log"
+	"github.com/Microsoft/hcsshim/internal/shimdiag"
+	"github.com/Microsoft/hcsshim/internal/uvm"
+	"github.com/pkg/errors"
+)
+
+func createPnPInstallDriverCommand(driverUVMPath string) []string {
+	dirFormatted := fmt.Sprintf("%s/*.inf", driverUVMPath)
+	args := []string{"cmd", "/c", "C:\\Windows\\System32\\pnputil.exe", "/add-driver", dirFormatted, "/subdirs", "/install", "&"}
```

Comment:
Can you spread this out over multiple lines like we do elsewhere? Makes it a little easier to follow
https://github.com/microsoft/hcsshim/blob/2ba2cb51c5da57bbdbcaf827ef0ed3469bda3697/test/cri-containerd/execcontainer_test.go#L102 for example.